components/AuthorizedPage/AuthorizedPage.tsx

```diff
@@ -0,0 +1,57 @@
+import { ReactNode, useEffect, useMemo, useState } from "react";
+import useApp from "../../hooks/useApp";
+import jwt from "jsonwebtoken";
+import { isInIframe } from "../../utils/misc";
+import { DashboardTokenPayload } from "../../lib/middlewares";
+import AccessWarning from "../AccessWarning/AccessWarning";
+import LoadingPage from "../LoadingPage/LoadingPage";
+
+type AuthorizedPageProps = {
+  children: ReactNode;
+};
+
+const AuthorizedPage = ({ children }: AuthorizedPageProps) => {
+  const app = useApp()?.getState();
```

Comment:
Generic question - why can't `useApp` return always defined value? If initially it's nullish, can't we return something else at app.tsx?